src/main/java/uk/gov/hmcts/reform/ccd/client/model/CallbackRequest.java

```diff
@@ -2,11 +2,15 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 @Data
 @Builder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
what's the reason for these? previously this client was built using a builder pattern why is the API being compromised by adding a 0 args constructor?